src/containers/Application/ApplicationSubmission.tsx

```diff
@@ -48,7 +48,7 @@ const ApplicationSubmission: React.FC = () => {
 
   useEffect(() => {
     if (!isApplicationLoaded || statusLoading) return
-    console.log('appStatus', appStatus)
+    // console.log('appStatus', appStatus)
```

Comment:
I think it would be better to remove this `console.log` too, so we don't end-up having many commented logs.